server/src/main/kotlin/suwayomi/tachidesk/manga/impl/update/UpdateStatus.kt

```diff
@@ -0,0 +1,35 @@
+package suwayomi.tachidesk.manga.impl.update
+
+import mu.KotlinLogging
+import suwayomi.tachidesk.manga.model.dataclass.MangaDataClass
+
+var logger = KotlinLogging.logger {}
+class UpdateStatus {
+    var statusMap = HashMap<JobStatus, ArrayList<MangaDataClass>>()
```

Comment:
```suggestion
    var statusMap = mutableMapOf<JobStatus, MutableList<MangaDataClass>>()
```
Avoid raw java types, unless you specifically need a non-standard function from them